README.md

```diff
@@ -170,5 +195,10 @@ Write what an example ERB file (aka view) might look like to list all the warrio
 
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
+<% @warriors.each do |warrior| %>
+<ul>
+<li><%=warrior%></li>
```

Comment:
We should use `warrior.name` here, as without it, we'd output not-useful info like the memory address where the warrior is in our computer. Here we just want the name of the warrior.